bin/checkout-artifacts.sh

```diff
@@ -21,7 +21,7 @@ if [ ! -d $ARTIFACTS_DIR/.git ]; then
 fi
 
```

Comment:
It may help to do `echo "Download test dependencies via git"` or something alike, as it may be surprising to see git in action when calling `./test`